WORKFLOW.md

```diff
@@ -0,0 +1,38 @@
+# Workflow
+
+**Note:** As much as possible, this workflow has to be follow when
+  contributing to GreenPix repositories. In exceptional cases,
+  specific workflow can be used and the project README or a dedicated
+  WORKFLOW file will describe it.
+
+Core Team members and regular Contributors (see ROLES.md) have write
+access to GreenPix repositories.  That is, they can push commit
+directly to those repositories. If you are a new contributor, you do
+not have write accesses. You have to fork the targeted repositories
+and push your changes into those forks.
+
+It's a good practice that even Core Team members work on their own fork
+for minor or trivial features and keep branches in upstream
```

Comment:
Yeah, I think we should try to start doing that ourselves. But I would add that this is not necessary for project that have just started. Only when the project has reach a state where it make sense. This could also be dictated by the number of person working on the project. As long as just one person is working on the project, she or he can work directly on the repo instead of working on a fork. I still think it's good practice to use the PR system as much as possible though.